mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -169,11 +173,10 @@ def ocn_modelvsobs(config, field):
 
         if buildObsClimatologies:
             dsObs = xr.open_mfdataset(obsFileName)
-
-            dsTimeSlice = dsObs.sel(time=slice(timeStart, timeEnd))
-            monthlyClimatology = dsTimeSlice.groupby('time.month').mean('time')
-
-            dsObs = monthlyClimatology.transpose('month', 'lat', 'lon')
+            dsObs.rename({'time': 'Time'}, inplace=True)
+            dsObs = dsObs.transpose('Time', 'lat', 'lon')
```

Comment:
These are pretty small data sets and we've been transposing them the whole time.  I just revised the code a bit.